docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,508 @@
+
+## Prerequisites
+
+### Operating system
+
+IoT Setup Manager supports the following operating systems:
+
+-   Windows 7 and later (64-bit and 32-bit)
+-   Ubuntu 14.04 LTS and later (64-bit and 32-bit)
+
+### Ubuntu tools
+
+If you are using Ubuntu, the following packages must be installed in addition to the IoT Setup Manager:
+
+-   rpm2cpio
+-   cpio
+
+If the packages are not installed, run the following command to install a package:
+
+```
+sudo apt-get install <package-name>
+```
+
+### Download binaries
+
+You must have the supported binary images in your computer. You can download the binary images from:
+
+1.   To Download images visit [Downloads](http://download.tizen.org/releases/milestone/tizen).
+2.   Click on **6.0-unified (or "unified" whichever is present) > latest > images > standard**.
+3.   For different devices or profiles download the compressed file from:
+
+     1.   RPI3 Headless 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi3/
+          2.   Platform Image: iot-headless-2parts-armv7l-btrfs-rootfs-rpi/
+
+     2.   RPI3 Headed 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi3/
+          2.   Platform Image: iot-headed-3parts-armv7l-rpi/
+
+     3.   RPI4 Headless 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi4/
+          2.   Platform Image: iot-headless-2parts-armv7l-btrfs-rootfs-rpi/
+
+     4.   RPI4 Headed 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi4/
+          2.   Platform Image: iot-headed-3parts-armv7l-rpi/
+
+     5.   RPI3 Headed 64-bit:
+          1.   Boot Image: iot-boot-arm64-rpi3/
+          2.   Platform Image: iot-headed-3parts-aarch64-rpi/
+
+     6.   RPI4 Headed 64-bit:
+          1.   Boot Image: iot-boot-arm64-rpi4/
+          2.   Platform Image: iot-headed-3parts-aarch64-rpi/
+
+## Install tizen studio
+
+>    [!NOTE]
+>
+>    You can skip this section, if you have already installed latest Tizen Studio 4.0 and installed the **6.0 Mobile** profile for the **Main SDK** and the **IoT Setup Manager**, **IoT-Headed-6.0** and **IoT-Headless-6.0** profiles for the **Extension SDK**.
+
+To install Tizen Studio and the required profiles for IoT development, follow these steps:
+
+1.  Download and install Tizen Studio 4.0 for your operating system from [Tizen developer site](https://developer.tizen.org/development/tizen-studio/download).
+2.  Open Tizen Studio and go to **Tools > Package Manager**.
+3.  Click **Extension SDK** and verify whether the IoT Setup Manager appears in the list of available packages.
+4.  Click **install** for **IoT Setup Manager** to install the IoT Setup Manager Extension.
+
+    ![Install the headless profile](media/package_manager_iot.png)
+	
+5.  Click **Main SDK** tab, and install the **6.0 Mobile** profile:
+
+    ![Install the mobile profile](media/package_manager_mobile.PNG)
+	
+    >    [!NOTE]
+    >
+    >    **Extension SDK** should be updated to the latest version.
```

Comment:
Earlier comment: https://github.com/Samsung/tizen-docs/pull/1293#discussion_r509315857

Please change to:

> [!NOTE]
> **Extension SDK** must be updated to the latest version.